checker/jdk/index/src/java/lang/String.java

```diff
@@ -162,7 +162,7 @@ public String() {
      * @param  original
      *         A {@code String}
      */
-    public @PolyMinLen @PolySameLen String(@PolyMinLen @PolySameLen String original) {
+    public /*@ PolyMinLen*/ @PolySameLen String(/*@ PolyMinLen*/ @PolySameLen String original) {
```

Comment:
`@ PolyMinLen` is the same as `@PolyMinLen`, so it's getting compiled into the code and causing the tests to fail.